packages/template/package.json

```diff
@@ -18,41 +18,47 @@
     "publish-storybook": "bash .scripts/publish_storybook.sh"
   },
   "devDependencies": {
-    "react": "^15.3.2",
-    "react-dom": "^15.3.2",
-
-    "babel-loader": "^6.2.5",
-    "babel-polyfill": "^6.13.0",
-    "babel-preset-react-app": "^0.2.1",
+    "@kadira/storybook": "^2.18.1",
     "babel-cli": "^6.14.0",
     "babel-core": "^6.14.0",
+    "babel-eslint": "^6.1.2",
+    "babel-loader": "^6.2.5",
     "babel-plugin-transform-runtime": "^6.15.0",
-
+    "babel-polyfill": "^6.13.0",
+    "babel-preset-react-app": "^0.2.1",
+    "chai": "^3.5.0",
+    "css-loader": "^0.25.0",
+    "enzyme": "^2.2.0",
     "eslint": "^3.6.0",
-    "babel-eslint": "^6.1.2",
     "eslint-config-airbnb": "^12.0.0",
     "eslint-plugin-import": "^1.16.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
-
+    "git-url-parse": "^6.0.1",
+    "jsdom": "^9.5.0",
     "mocha": "^3.0.2",
-    "chai": "^3.5.0",
-    "sinon": "^1.17.6",
-    "enzyme": "^2.2.0",
+    "react": "^15.3.2",
     "react-addons-test-utils": "^15.3.2",
-    "jsdom": "^9.5.0",
-
-    "@kadira/storybook": "^2.18.1",
-    "git-url-parse": "^6.0.1"
+    "react-dom": "^15.3.2",
+    "sinon": "^1.17.6"
   },
   "peerDependencies": {
     "react": "^0.14.7 || ^15.0.0"
   },
   "dependencies": {
-    "babel-runtime": "^6.11.6"
+    "babel-runtime": "^6.11.6",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "uniforms": "^1.5.0-rc.1",
+    "uniforms-bootstrap4": "^1.5.0-rc.1"
```

Comment:
@zeroasterisk 

* changes: moved everything except `bable-runtime` to `peerDependencies`